packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -125,32 +125,20 @@ <h2>Registrazione</h2>
       <div>
         In data <strong>{{submissionDate}}</strong> alle ore
         <strong>{{submissionTime}}</strong>, l’Infrastruttura ha trasmesso
-        all’erogatore la richiesta di fruizione.
+        {{#if producerDelegationId}} al delegante all’erogazione{{else}} all’erogatore{{/if}} la richiesta di fruizione.
       </div>
-      {{#if producerDelegationId}}
       <div>
         In data <strong>{{activationDate}}</strong> alle ore
         <strong>{{activationTime}}</strong>, l’infrastruttura ha ricevuto
         la dichiarazione di accettazione della richiesta di fruizione
-        (di seguito “dichiarazione di accettazione”) inviata dal delegato all’erogazione,
+        (di seguito “dichiarazione di accettazione”) inviata {{#if producerDelegationId}}dal delegato all’erogazione,{{else}}inviata dall’erogatore,{{/if}}
         tramite l’operatore amministrativo con identificativo Area Riservata
-        <strong>{{producerDelegateId}}</strong>.
-      </div>
-      {{else}}
-      <div>
-        In data <strong>{{activationDate}}</strong> alle ore
-        <strong>{{activationTime}}</strong>, l’Infrastruttura ha ricevuto la
-        dichiarazione di accettazione della richiesta di fruizione (di seguito
-        “dichiarazione di accettazione”) inviata dall’erogatore, tramite
-        l’operatore amministrativo con identificativo Area Riservata
-        <strong>{{activatorId}}</strong>.
+        {{#if producerDelegationId}}<strong>{{producerDelegateId}}</strong>.{{else}}<strong>{{activatorId}}</strong>.{{/if}}
```

Comment:
Sorry, another thing I missed in reviewing the previous PR 👇 
This is always the `activatorId` in both cases, it cannot be the delegateId because they are two different things semantically: the activatorId is a UserId while the delegateId is a TenantId.

```suggestion
        <strong>{{activatorId}}</strong>.
```

Reply:
Tahnk @ecamellini  good catch ! 
soleved in [4a06701](https://github.com/pagopa/interop-be-monorepo/pull/1243/commits/4a0670156e0fea3822531129e92161195b9124a4)
Merge state
This branch has not been deployed
No deployments
Changes requested
1 review requesting changes by reviewers with write access. 
1 change requested
5 pending reviewers
Some checks haven’t completed yet
48 in progress, 2 successful, and 1 expected checks
@github-actions
CodeQL / Analyze (javascript-typescript) (dynamic) In progress — This check has started...
Details
@github-actions
PR validation / Check formatting (pull_request) In progress — This check has started...
Required
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035247/job/33797609076?pr=1243)
@github-actions
PR validation / Lint (pull_request) In progress — This check has started...
Required
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797609081?pr=1243)
@github-actions
PR validation / Check (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797610371?pr=1243)
@github-actions
PR validation / Test (pull_request) In progress — This check has started...
Required
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797609481?pr=1243)
@github-actions
PR validation / Find Dockerfiles (pull_request) Successful in 3s
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797610915?pr=1243)
@github-actions
PR validation / Build (purpose-outbound-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797609923?pr=1243)
@github-actions
PR validation / Build (eservice-descriptors-archiver) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797618973?pr=1243)
@github-actions
PR validation / Build (catalog-platformstate-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797620213?pr=1243)
@github-actions
PR validation / Build (agreement-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797620977?pr=1243)
@github-actions
PR validation / Build (catalog-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797621471?pr=1243)
@github-actions
PR validation / Build (authorization-platformstate-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797621877?pr=1243)
@github-actions
PR validation / Build (backend-for-frontend) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797622583?pr=1243)
@github-actions
PR validation / Build (purpose-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797623040?pr=1243)
@github-actions
PR validation / Build (tenant-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797623465?pr=1243)
@github-actions
PR validation / Build (authorization-updater) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797624504?pr=1243)
@github-actions
PR validation / Build (ivass-certified-attributes-importer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797625628?pr=1243)
@github-actions
PR validation / Build (agreement-platformstate-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797627131?pr=1243)
@github-actions
PR validation / Build (key-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797628664?pr=1243)
@github-actions
PR validation / Build (agreement-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797629171?pr=1243)
@github-actions
PR validation / Build (dtd-catalog-exporter) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797629582?pr=1243)
@github-actions
PR validation / Build (client-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797630066?pr=1243)
@github-actions
PR validation / Build (catalog-outbound-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797630750?pr=1243)
@github-actions
PR validation / Build (tenant-outbound-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797631200?pr=1243)
@github-actions
PR validation / Build (notifier-seeder) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797631603?pr=1243)
@github-actions
PR validation / Build (api-gateway) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797631991?pr=1243)
@github-actions
PR validation / Build (token-details-persister) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797632432?pr=1243)
@github-actions
PR validation / Build (compute-agreements-consumer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797632844?pr=1243)
@github-actions
PR validation / Build (agreement-email-sender) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797633383?pr=1243)
@github-actions
PR validation / Build (agreement-outbound-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797633826?pr=1243)
@github-actions
PR validation / Build (one-trust-notices) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797634243?pr=1243)
@github-actions
PR validation / Build (tenant-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797634708?pr=1243)
@github-actions
PR validation / Build (authorization-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797635157?pr=1243)
@github-actions
PR validation / Build (pn-consumers) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797635607?pr=1243)
@github-actions
PR validation / Build (delegation-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797636050?pr=1243)
@github-actions
PR validation / Build (catalog-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797636567?pr=1243)
@github-actions
PR validation / Build (delegation-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797636952?pr=1243)
@github-actions
PR validation / Build (authorization-server) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797637356?pr=1243)
@github-actions
PR validation / Build (anac-certified-attributes-importer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797637803?pr=1243)
@github-actions
PR validation / Build (datalake-interface-exporter) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797638306?pr=1243)
@github-actions
PR validation / Build (attribute-registry-process) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797638751?pr=1243)
@github-actions
PR validation / Build (producer-key-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797639228?pr=1243)
@github-actions
PR validation / Build (attribute-registry-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797639740?pr=1243)
@github-actions
PR validation / Build (producer-key-events-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797640147?pr=1243)
@github-actions
PR validation / Build (purpose-platformstate-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797640526?pr=1243)
@github-actions
PR validation / Build (purpose-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797640960?pr=1243)
@github-actions
PR validation / Build (selfcare-onboarding-consumer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797641388?pr=1243)
@github-actions
PR validation / Build (datalake-data-export) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797641874?pr=1243)
@github-actions
PR validation / Build (producer-keychain-readmodel-writer) (pull_request) In progress — This check has started...
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797642485?pr=1243)
Build Result Expected — Waiting for status to be reported
Required
@dpulls
dpulls — All dependencies are resolved.
[Details](https://github.com/pagopa/interop-be-monorepo/actions/runs/12123035538/job/33797642950?pr=1243)
This branch is out-of-date with the base branch
Merge the latest changes from develop into this branch.
This merge commit will be associated with vittorio.caprio@buildo.io.

 You can also open this in GitHub Desktop or view .
✨ 
@Viktor-K


Add a comment
Comment
 
Add your comment here...
 
Remember, contributions to this repository should follow our GitHub Community Guidelines.
 ProTip! Add comments to specific lines under Files changed.
Reviewers
@ecamellini
ecamellini

@beetlecrunch
beetlecrunch

@galales
galales

@AsterITA
AsterITA

@Carminepo2
Carminepo2

@taglioni-r
taglioni-r

Requested changes must be addressed to merge this pull request.

Still in progress?
Assignees
No one—
Labels
None yet
Projects
None yet
Milestone
No milestone
Development
Successfully merging this pull request may close these issues.

None yet

Loading
2 participants
@Viktor-K
@[ecamellini](https://github.com/ecamellini)